lib/artemis_api/client.rb

```diff
@@ -27,12 +27,16 @@ def find_one(type, id, facility_id: nil, include: nil, force: false)
       if !obj || force
         refresh if @oauth_token.expired?
 
-        url = if facility_id
-                "#{@options[:base_uri]}/api/v3/facilities/#{facility_id}/#{type}/#{id}"
-              else
-                "#{@options[:base_uri]}/api/v3/#{type}/#{id}"
-              end
-        url = "#{url}?include=#{include}" if include
+        path = if facility_id
+                 "/api/v3/facilities/#{facility_id}/#{type}/#{id}"
+               else
+                 "/api/v3/#{type}/#{id}"
+               end
+
+        query = ''
+        query += "include=#{include}" if include
+
+        url = build_url(path: path, query: query)
```

Comment:
```rb
query = {}
query[:include] = include if include
```

`query: URI.encode_www_form(query)` (if needed)

Reply:
this works here, but I can't figure out how to do it below where i'm dealing with filters? 

`query[:anything] = value` formats it like `?anything=value` but with filters it actually needs to be like.... `?filter[anything]=value` and im not sure how to achieve that with this method?